bin/ember-template-lint.js

```diff
@@ -83,7 +82,14 @@ function run() {
   var exitCode = 0;
 
   var configPath = checkConfigPath();
-  var linter = new Linter({ configPath });
+  var linter;
+  try {
+    linter = new Linter({ configPath });
+  } catch (e) {
```

Comment:
Sorry I missed this in review, I don't understand why we need to `console.error(e.message)` and `process.exit()` here. If `new Linter({ configPath })` throws an error the **default** thing is to log that error to the console and exit the process, right?

Reply:
As far as I remember, I thought the user would be better off with a simpler (more understandable, less technical?) output message than with the raw output from Node's error handler.

That is debatable but maybe then it just raises the question of error management in general. YMMV.